evap/student/templates/student_vote.html

```diff
@@ -46,14 +46,14 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 <div class="card-header">
                     <span class="fas fa-exclamation-triangle"></span> {% trans 'Small number of participants' %}
                 </div>
-                <div class="card-body">
+                <div class="card-body" id="few-participants-warning">
```

Comment:
Hm, if we change this id, `#text_results_publish_confirmation_top` and friends should also be renamed. But I’m not quite sure whether you really need this id.

Reply:
You are right, this was a leftover :)